source/Symbol/SwiftASTContext.cpp

```diff
@@ -737,7 +737,8 @@ SwiftEnumDescriptor::CreateDescriptor(swift::ASTContext *ast,
       elements_with_payload = enum_impl_strategy.getElementsWithPayload();
   llvm::ArrayRef<swift::irgen::EnumImplStrategy::Element>
       elements_with_no_payload = enum_impl_strategy.getElementsWithNoPayload();
-  if (enum_decl->isResilient())
+  swift::SILType swift_sil_type = irgen_module.getLoweredType(swift_can_type);
+  if (!irgen_module.getTypeInfo(swift_sil_type).isFixedSize())
```

Comment:
I would feel more comfortable with the change if we could move this case further down, since it's a bit of a catch-all. Are we sure all of the remaining cases have a fixed-size typeinfo?

Reply:
Should we put it as _first_ case instead? Going to IRGen without a fixed type is basically a recipe for trouble.

Reply:
I agree with 

> Going to IRGen without a fixed type is basically a recipe for trouble.

but my question remains: does non-fixed-type always imply resilient?